warp/src/main/java/com/schibsted/nmp/warp/components/WarpButton.kt

```diff
@@ -70,10 +78,18 @@ fun WarpButton(
         buttonStyle = buttonStyle,
         loading = loading
     ) {
+        icon?.let {
+            Icon(
+                modifier = Modifier
+                    .padding(end = dimensions.space1),
+                imageVector = ImageVector.vectorResource(icon),
+                contentDescription = iconContentDescr
```

Comment:
In FINN legendary we also have a possibility to change tint of the icon and icon placement can be at the end also
```
data class IconConfig(
    val iconResource: Int,
    val iconPlacement: IconPlacement = IconPlacement.Start,
    val iconPadding: Dp = ButtonDefaults.IconSpacing,
    val description: String? = null,
    val tint: Color? = null
)
```

Reply:
I intentionally left out tint so that the button can do it depending on which style it has. That way you will get correct brand colors for night and dark mode.
I will add the placement option!